### PR TITLE
bugfix/add-default-limit-to-noripple_check

### PR DESCRIPTION
**Pull request by officialfrancismendoza**
**Issue** (#223): No limit included in return when invoking noripple_check API
**Fix**:  Add default limit to noripple_check

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -121,3 +121,4 @@ Existing maintainers can resign, or be subject to a vote for removal at the behe
 * [legleux](https://github.com/legleux) (Ripple)
 * [undertome](https://github.com/undertome) (Ripple)
 * [godexsoft](https://github.com/godexsoft) (Ripple)
+* [officialfrancismendoza](https://github.com/officialfrancismendoza) (Ripple)
```
Comment (godexsoft) [inline CONTRIBUTING.md]: Not sure how this change got into this pr but it should not be (it was in #285 already)

File: src/backend/BackendInterface.h [hunk 2]
```diff
@@ -10,6 +10,10 @@
 
 namespace Backend {
 
+/**
```
Comment (godexsoft) [inline src/backend/BackendInterface.h]: All these doxy comments should probably not be part of this pr either. Basically all changes but the limit line should not be in this pr.

File: src/rpc/RPC.cpp [hunk 3]
```diff
@@ -225,7 +225,7 @@ static HandlerTable handlerTable{
     {"account_offers", &doAccountOffers, LimitRange{10, 50, 256}},
     {"account_tx", &doAccountTx, LimitRange{1, 50, 100}},
     {"gateway_balances", &doGatewayBalances, {}},
-    {"noripple_check", &doNoRippleCheck, {}},
+    {"noripple_check", &doNoRippleCheck, LimitRange{1, 300, 500}},
```
Comment (godexsoft) [inline src/rpc/RPC.cpp:267]: This is the only change to leave in this pr. Remove every other change please.

**Review by godexsoft — CHANGES_REQUESTED** [review]

**Review by cjcobb23 — CHANGES_REQUESTED** [review]
I agree with @godexsoft , this should be a one-line change in a single commit

**Comment by officialfrancismendoza** [comment]
@godexsoft found a great hotfix. All I have to do is run `git reset --hard origin/develop` to eliminate all the egregious commits and be up-to-date with latest develop commits, make changes to files of interest, and then rebase that way

**Review by godexsoft — DISMISSED** [review]
LGTM!

**Review by cjcobb23 — DISMISSED** [review]
LGTM

**Comment by officialfrancismendoza** [comment]
@godexsoft @cjcobb23 I do not have privileges to merge PRs to develop
image

**Comment by godexsoft** [comment]
> @godexsoft @cjcobb23 I do not have privileges to merge PRs to develop image

Looks like you forgot to sign (gpg) your commit. If unsure - go check CONTRIBUTING.md for details :+1:

**Review by cjcobb23 — DISMISSED** [review]
LGTM

**Review by cjcobb23 — APPROVED** [review]

**Pull request merged**